src/pages/components/menu.mdx

```diff
@@ -132,11 +131,57 @@ import MenuMediaCode from '!!raw-loader!../../examples/dropdowns/MenuMedia.tsx';
 
 ## Configuration
 
-<Configuration reactPackage={props.data.mdx.package} propSheets={props.data.mdx.propsSheets} />
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
 
 ## API
 
-<PropSheets propSheets={props.data.mdx.propsSheets} />
+### Menu
+
+<PropSheet components={props.data.mdx.components} componentName="menu" />
+
+### Separator
+
+<PropSheet components={props.data.mdx.components} componentName="separator" />
+
+### AddItem
+
+<PropSheet components={props.data.mdx.components} componentName="addItem" />
+
+### HeaderIcon
+
+<PropSheet components={props.data.mdx.components} componentName="headerIcon" />
+
+### HeaderItem
+
+<PropSheet components={props.data.mdx.components} componentName="headerItem" />
+
+### Item
+
+<PropSheet components={props.data.mdx.components} componentName="item" />
+
+### ItemMeta
```

Comment:
Some components don't have additional props besides their native backing element attributes, for example [ItemMeta](https://5ee914ebd759f501f95844d8--zendeskgarden.netlify.app/components/menu#itemmeta), MediaBody, MediaFigure, Separator, etc. Should the copy eventually be something along the lines of "Accepts all `<div>` props"?

<img width="1040" alt="Screen Shot 2020-06-16 at 12 53 01 PM" src="https://user-images.githubusercontent.com/1811365/84821686-59a0f300-afd0-11ea-91e6-b5bffc0cbe2b.png">

Reply:
I really hope not re: "Accepts all `<xxx>` props". We need to DRY up redundant copy and add it to the component overview page. But per **Discussion** mentioned in PR description, the lacking description paragraphs need design consideration. In the end, every single component needs a helpful high-level description.

Reply:
I think this is a great opportunity for us to write one or two line utility descriptors for these components (not now, but soon). I personally find myself looking at these components and then scrolling up to examples to determine how they should be used. Could be nice to try to explain the utility here to help eliminate that back and forth.